AccountsManager/client/src/main/java/accounts/AccountsClient.java

```
@@ -0,0 +1,86 @@
+package accounts;
+
+
+import accounts.pojos.AccountData;
+import accounts.pojos.CreateAccountRequest;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import exceptions.DbAccessException;
+
+import javax.ws.rs.InternalServerErrorException;
+import javax.ws.rs.NotAuthorizedException;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+import javax.xml.bind.DataBindingException;
+import java.io.IOException;
+import java.util.Optional;
+
+public class AccountsClient {
+
+    private final String accountsManagerBaseUri;
+    ObjectMapper mapper = new ObjectMapper();
+
+
+    public AccountsClient(String accountsManagerBaseUri) {
+        this.accountsManagerBaseUri = accountsManagerBaseUri;
+    }
+
+    public Optional<AccountData> getAccount(String accountToken) throws DbAccessException {
+        Client client = ClientBuilder.newClient();
+        String getAccountUri = accountsManagerBaseUri + "/accounts";
+        Response response = client.target(getAccountUri)
+                .request().header("X-ACCOUNT-TOKEN", accountToken).get();
+
+        if (response.getStatusInfo().equals(Response.Status.UNAUTHORIZED)) {
+            return Optional.ofNullable(null);
+        }
+        else if (!response.getStatusInfo().equals(Response.Status.OK)) {
+            throw new DbAccessException("Couldn't retrieve account");
```

Why Db? You are in a client context
You can simply throw a runtime exception